File: ipc2-ejemplos-2021--mas-reportes-ejemplos/calc-app-ui-mas-reportes-ejemplos/src/objects/calculator/Historial.ts
import { OperatorEnum } from './OperatorEnum';
export class Historial {

    idHistorial: Number;
    numero1: Number;
    numero2: Number;
    operacion: OperatorEnum;
    resultado: string;



    constructor(idHistorial: Number, numero1: Number, numero2: Number, operacion: OperatorEnum, resultado: string) {
        this.idHistorial = idHistorial;
        this.numero1 = numero1;
        this.numero2 = numero2;
        this.operacion = operacion;
        this.resultado = resultado;
    }

    get getidHistorial() {
      return this.idHistorial
    }
    
    set setidHistorial(val: Number) {
      this.idHistorial = val
    }
    
    get getnumero1() {
      return this.numero1
    }
    
    set setnumero1(val: Number) {
      this.numero1 = val
    }
    
    get getnumero2() {
      return this.numero2
    }
    
    set setnumero2(val: Number) {
      this.numero2 = val
    }
    
    get getoperacion() {
      return this.operacion
    }
    
    set setoperacion(val: OperatorEnum) {
      this.operacion = val
    }
    
    get getresultado() {
      return this.resultado
    }
    
    set setresultado(val: string) {
      this.resultado = val
    }

    

}

File: ipc2-ejemplos-2021--mas-reportes-ejemplos/calc-app-api-mas-reportes-ejemplos/src/main/java/com/jgranados/calcappapi/operations/domain/Historial.java
package com.jgranados.calcappapi.operations.domain;

/**
 *
 * @author jose
 */
public class Historial {

    private int idHistorial;
    private int numero1;
    private int numero2;
    private Operation operacion;
    private String resultado;

    public int getNumero1() {
        return numero1;
    }

    public void setNumero1(int numero1) {
        this.numero1 = numero1;
    }

    public int getNumero2() {
        return numero2;
    }

    public void setNumero2(int numero2) {
        this.numero2 = numero2;
    }

    public Operation getOperacion() {
        return operacion;
    }

    public void setOperacion(Operation operacion) {
        this.operacion = operacion;
    }

    public String getResultado() {
        return resultado;
    }

    public void setResultado(String resultado) {
        this.resultado = resultado;
    }

    public int getIdHistorial() {
        return idHistorial;
    }

    public void setIdHistorial(int idHistorial) {
        this.idHistorial = idHistorial;
    }
}
